config/locales/server.en.yml

```diff
@@ -1882,6 +1882,9 @@ en:
     max_mentions_per_post: "Maximum number of @name notifications anyone can use in a post."
     max_users_notified_per_group_mention: "Maximum number of users that may receive a notification if a group is mentioned (if threshold is met no notifications will be raised)"
     enable_mentions: "Allow users to mention other users."
+    here_mention: "Name used for @here mention. Must not be an existent username."
```

Comment:
Ditto: https://github.com/discourse/discourse/pull/14900/files#r751821285

Reply:
This is the translation string for the site setting that controls what "@here" will be. I think this one should stay this way.